### PR TITLE
fix: remove fifo path if stdio is not initail

### PR DESCRIPTION
**Pull request by zhuangqh**
### Ⅰ. Describe what this PR did
remove fifo path if stdio is not initail

### Ⅱ. Does this pull request fix one issue?

NONE

### CODE REVIEW

**Comment by Ace-Tang** [comment]
Just for the logic, LGTM

File: daemon/containerio/cio.go [hunk 1]
```diff
@@ -198,6 +198,10 @@ func WithAttach(stdin io.Reader, stdout, stderr io.Writer) containerdio.Attach {
 		if paths == nil {
 			return nil, fmt.Errorf("cannot attach to existing fifos")
 		}
+		// judge stdin is initial, make logic same with copyIO.
+		if i, ok := stdin.(*ContainerIO); (ok && i == nil) || (stdin == nil) {
+			paths.In = ""
+		}
```
Comment (Ace-Tang) [inline daemon/containerio/cio.go]: I just wonder if this code is need here, code not so good.

Comment (zhuangqh) [reply daemon/containerio/cio.go]: @Ace-Tang What did you concern about? Maybe we should use `reflect.ValueOf(stdin).IsNil()` to check whether stdin is nil?

**Comment by zhuangqh** [comment]
There is a mechanism to promise `paths.In == ""` if stdin is not initial. closed. @Ace-Tang

**Comment by allencloud** [comment]
LGTM

**Pull request merged**